test/parallel/test-process-execpath.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const child_process = require('child_process');
+const path = require('path');
+const fs = require('fs');
+
+assert.strictEqual(process.execPath, fs.realpathSync(process.execPath));
+
+if (process.argv[2] === 'child') {
+  console.log(process.execPath);
```

Comment:
can you please add a comment mentioning at the the `console.log()` output here is part of the test